cluster/gce/gci/configure-helper.sh

```diff
@@ -2811,6 +2811,32 @@ function start-lb-controller {
   fi
 }
 
+###start cluster networking: flannel, bridge
+function start-cluster-networking {
+  case "${NETWORK_POLICY_PROVIDER:-flannel}" in
+    flannel)
+    start-flannel-ds
+    ;;
+    bridge)
+    start-bridge-networking
+    ;;
+  esac
+}
+
+function start-flannel-ds {
+  if [[ -f "${KUBE_HOME}/flannel/kube-flannel.yml" ]]; then
+    echo "installing flannel ds"
+    kubectl apply -f "${KUBE_HOME}/flannel/kube-flannel.yml"
```

Comment:
how we can keep this yaml file updated if some flannel binary/image changes ?

Reply:
this yaml file is downloaded from kube-flannel in cluster/gce/gci/configure.sh, it is always the latest version.
```
function install-flannel-yml {
  echo "downloading flannel"
  download-or-bust "" "https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml"
  local -r flannel_dir="${KUBE_HOME}/flannel"
  mkdir -p "${flannel_dir}"
  mv "${KUBE_HOME}/kube-flannel.yml" "${flannel_dir}"
}
```